server/app.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const port = 3000;
+const db = require('../database/index.js');
+
+app.use(cors());
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(bodyParser.json());
+app.use(bodyParser.raw());
+app.use(express.static(__dirname + '/public'));
+
+app.get('/games', (req, res) => {
+  const getAllGames = db.getAllGames();
+  getAllGames.then((games) => {
+    res.send(games);
+  })
+});
+
+app.listen(port, () => {
```

Comment:
Nit: No need to include port as an argument - it's defined it in a broader scope and written in a template literal.